Dockerfile

```diff
@@ -62,7 +61,7 @@ RUN echo "deb http://ftp.debian.org/debian $(sed -n 's/^VERSION=.*(\(.*\)).*/\1/
     # fully qualified domain name
     && echo 'ServerName localhost' > /etc/apache2/conf-available/fqdn.conf \
     # Grab and install wp-cli from remote
-    && curl \
+    && curl --create-dirs \
```

Comment:
In some of the newer base images it appears `/etc/bash_completion.d` isn't available (prob due to bash completion being offloaded or something). So `--create-dirs` fixes that issue.